engine/action_contact.go

```diff
@@ -403,6 +403,48 @@ func (c *Client) upsertContact(ctx context.Context, paymailService paymailclient
 	return contact, nil
 }
 
+// AdminConfirmContacts confirms provided contacts, should be used only by the admin.
+func (c *Client) AdminConfirmContacts(ctx context.Context, contacts []*Contact) error {
+	contactList := []*Contact{}
```

Comment:
Please preallocate memory for this slice - because we already know the final size.

Reply:
Size preallocated